### PR TITLE
Filtering backend

### PR DESCRIPTION
**Pull request by KennaReed**
Modified existing functionality to handle the backend for filtering. Files to review: postBackend.py and model_mongodb.py

### CODE REVIEW

**Review by dathash — APPROVED** [review]
Looks great! Merge requests are pretty messy with all these random dependency files though, I wonder if there is a solution. Anyway, not much to change.

**Review by jackson14furie — APPROVED** [review]
It looks good to me. A little hard to find the files I want with all of the random files we have but  I found it eventually. Let me know if I'm doing this wrong or anything

**Review by billc2501 — COMMENTED** [review]
I like how neat and small these changes are in this commit

**Comment by billc2501** [comment]
I think most of the changes seemed necessary and there weren't any specific piece of code that is redundant. Besides the dependency aspects, this request seems clean.

**Review by billc2501 — APPROVED** [review]

File: postBackend.py [hunk 1]
```diff
@@ -55,6 +36,29 @@ def get_posts():
         resp = jsonify(newPost), 201
         return resp
 
+@app.route('/filter', methods=['POST'])
```
Comment (areed08) [inline postBackend.py:39]: No comments and looks good. The way you are accepting data from the front end should match how I am sending you the data. This code is clean :)

**Review by areed08 — APPROVED** [review]
Looks good. The way the backend is handling what I send to it should work out. A very solid commit that will hopefully fix my code.

**Pull request merged**